contracts/svg/template/halo.partial.hbs

```diff
@@ -0,0 +1,21 @@
+<defs>
+  <path
+    d="M58.7687 76.7949C110.299 88.0702 154.043 62.7974 190 0.976313L110 139.54C77.1838 120.42 37.9799 109.915 -3.21719e-05 110.066L-1.0516e-05 70.0661C19.718 70.0595 39.5625 72.3317 58.7687 76.7949Z"
+    fill="#9A9EA7"
+    id="h0"
+  ></path>
+  <circle cx="100" cy="-50" r="20" fill="#9A9EA7" id="h1"></circle>
+</defs>
+
+<g transform="translate(1000 1060)">
+  {{#each rhythm}}
+    <g
+      style="transform: rotate(calc({{uint2str
```

Comment:
this is really elegant, could we modify this to support some of the halos that have irregular gaps and the halos that take up more degrees with their shape? visual aid -> https://www.figma.com/file/xhR6TRIfuTEFb8aOdl87IH/wand-dev-and-launch?node-id=175%3A1307

Reply:
This should work already with the current approach, which divides the circle into 24 segments and then lets you toggle any combination of basic shapes for each segment. Looking at your catalogue I think we can even simplify it a bit, since we never combine different basic shapes in any one halo.

So I imagine any kind of halo can be represented in terms of the following information:

- basic shape ID: `0`-`n`
- rhythm:  `boolean[24]`

Each boolean rhythm item defines whether the basic shape shall be drawn on the respective circle segment or not.

A basic shape can also be larger than a single circle segment, we'd then only toggle it on every second segment for example.

The question is if dividing into 24 segments provides sufficient resolution for all the halos we wanna have. If not, it's no problem to divide into more segments of course.

Reply:
This is so cool